### PR TITLE
Enhancement/new structure sys

### PR DESCRIPTION
**Pull request by Raycoms**
Setup new structure system after structurize backend change

Review please

### CODE REVIEW

File: build.properties [hunk 1]
```diff
@@ -13,4 +13,4 @@ tinker_version=2.10.0.+
 dynamic_tree_version=0.7.+
 jei_version=4.8.5.147
 c_b_version=14.0.128
-structurize_version=0.10.47-ALPHA
+structurize_version=0.10.58-ALPHA
```
Comment (Nightenom) [inline build.properties]: 0.10.59 is the latest now, also you uploaded libs folder

File: src/main/java/com/minecolonies/coremod/entity/ai/util/Structure.java [hunk 2]
```diff
@@ -194,7 +194,7 @@ private static boolean structureBlockEqualsWorldBlock(
      * The internal structure loaded.
      */
     @Nullable
-    private final StructureWrapper theStructure;
+    private final com.ldtteam.structures.helpers.Structure theStructure;
```
Comment (Nightenom) [inline src/main/java/com/minecolonies/coremod/entity/ai/util/Structure.java]: Probably import this in the entire file

**Review by Nightenom — CHANGES_REQUESTED** [review]
Don't forget to remove libs/* from commit :P

File: src/main/java/com/minecolonies/coremod/entity/ai/util/Structure.java (hunk 2 above)
Comment (Raycoms) [reply src/main/java/com/minecolonies/coremod/entity/ai/util/Structure.java]: It had the same name as this file that's why it didn't work, I renamed it to distinguish better

File: build.properties (hunk 1 above)
Comment (Raycoms) [reply build.properties]: done

**Comment by Raycoms** [comment]
Removed the lib as well

**Review by Nightenom — DISMISSED** [review]

**Review by marchermans — DISMISSED** [review]

**Review by marchermans — APPROVED** [review]

**Pull request merged**